File: app/src/main/assets/templates/java-app/app/src/main/java/MainActivity.java
package {{PACKAGE_NAME}};

import android.app.Activity;
import android.graphics.Color;
import android.os.Bundle;
import android.view.Gravity;
import android.widget.TextView;

/**
 * A minimal starting screen — edit this and make it yours.
 * Everything here is plain Android, no extra dependencies.
 */
public class MainActivity extends Activity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);

        TextView welcome = new TextView(this);
        welcome.setText("Hello from {{APP_NAME}}!");
        welcome.setTextColor(Color.DKGRAY);
        welcome.setTextSize(24f);
        welcome.setGravity(Gravity.CENTER);
        setContentView(welcome);
    }
}

File: app/src/main/java/com/enajid/apkbuilder/MainActivity.kt
package com.enajid.apkbuilder

import android.os.Bundle
import androidx.activity.ComponentActivity
import androidx.activity.compose.setContent
import androidx.activity.enableEdgeToEdge
import com.enajid.apkbuilder.ui.ApkBuilderRoot
import com.enajid.apkbuilder.ui.theme.ApkBuilderTheme

class MainActivity : ComponentActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        enableEdgeToEdge()
        setContent {
            ApkBuilderTheme {
                ApkBuilderRoot()
            }
        }
    }
}
